File: loianeTraining/javaBasico/Exer Aula 16_17/Ex1.java
import java.util.Scanner;

class Ex1 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        var note = 0;
        do {
            System.out.println("Digite uma nota (0 - 10)");
            note = scan.nextInt();

            if (note < 0 || note > 10) {
                System.out.println("Valor invalido");
            }
        } while (note < 0 || note > 10);
        scan.close();
    }
}

File: Loiane_Training_C#/Basico/D/Ex1.cs
using System;

class Ex1
{
    static void Main(string[] args)
    {
        bool repeat = true;

        while (repeat)
        {
            Console.Write("Digite uma nota entre 0 e 10: ");
            int nota = int.Parse(Console.ReadLine());

            if (nota >= 0 && nota <= 10)
            {
                repeat = false;
                break;
            }
            else
            {
                Console.WriteLine("Valor invalido!");
            }
        }
    }
}